kuma/static/js/highlight.js

```diff
@@ -0,0 +1,16 @@
+(function(win, doc, $) {
+    'use strict';
+
+    // find all targets and wrap appropriately
+    function highlight(targets) {
+        var $targets = $(targets);
+        $targets.each(function() {
+            $(this).addClass('highlight-spanned').wrapInner('<span class="highlight-span"></span>');
+        });
+    }
+
+    // call on aritcle body with h3 and h4 as targets
+    var $articleSubHeads = $('#wikiArticle h3, #wikiArticle h4');
+    highlight($articleSubHeads);
+
+})(window, document, jQuery);
```

Comment:
Both js and no-js should have a black background and white text. The difference between the js and no-js experience is that once the classes have been applied the black background will not extend to the end of the line.

With JS:

![screen shot 2017-07-10 at 10 27 41](https://user-images.githubusercontent.com/854701/28030929-7b5593c0-655a-11e7-83cc-e6cb85e9bd86.png)

Without JS:

![screen shot 2017-07-10 at 10 27 53](https://user-images.githubusercontent.com/854701/28030939-80302180-655a-11e7-9dd2-ff0522bf4fc2.png)

If you're not seeing the JS version properly you probably have to do a `docker-compose restart web` since changes to `common.py` aren't handled without a restart.